beaumont/config_HOF_2021.json

```diff
@@ -0,0 +1,188 @@
+{
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w"
+      }
+    },
+    {
```

Comment:
You can add a second T1w with customLabels "rec-norm" to extract:
```
"ImageType": ["ORIGINAL", "PRIMARY", "M", "ND", "NORM"]
```
then you need to add to extract the original T1w:
```
   "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"],
```